app/assets/stylesheets/articles.scss

```diff
@@ -395,60 +163,38 @@
           }
         }
       }
-      .article-organization-headline {
-        display: block;
-        margin-bottom: calc(-1px - 0.6vw);
-        display: flex;
-        .article-organization-headline-inner {
-          @include themeable(
-            background,
-            theme-container-accent-background,
-            darken($light-gray, 6%)
-          );
-          @include themeable(color, theme-color, $black);
-          padding: 6px 12px;
-          border-radius: 100px;
-          font-weight: 600;
-          display: inline-block;
-          margin-top: 10px;
-          margin-left: 10px;
-          font-size: calc(0.9em);
-          img {
-            height: calc(1.3em);
-            width: calc(1.3em);
-            vertical-align: -0.27em;
-            margin-right: 6px;
-            border-radius: 100px;
-          }
-        }
-        a.org-headline-filler {
-          flex: 1;
+
+      // Organization Badge
+      .org-headline-filler {
+        @include themeable(background, theme-container-accent-background, darken($light-gray, 6%));
+        @include themeable(color, theme-color, $black);
+        padding: 8px;
+        border-radius: 3px 0 3px 0;
+        font-weight: 600;
+        display: inline-flex;
+        align-items: center;
+        font-size: 0.9em;
+        line-height: 1;
+        img {
+          width: 24px; 
+          height: 24px;
+          margin-right: 4px;
+          border-radius: 50%;
+          vertical-align: middle;
         }
       }
+
       .small-pic {
         float: left;
         height: 40px;
         width: 40px;
-        margin-left: 12px;
-        margin-top: 3px;
-        border-radius: 50px;
-        padding: 8px 0px 11px;
-        @media screen and (min-width: 430px) {
-          height: 50px;
-          width: 50px;
-          padding: 20px 0px 23px;
-        }
-        @media screen and (min-width: 1400px) {
-          padding: 23px 0px 27px;
-        }
+        margin: 16px 0 0 16px;
+        border-radius: 50%;
         img,
-        .color {
+        .color { // TODO: what is .color for?
```

Comment:
I don't think it's used based on what I saw in the codebase, but if you're not positive, just leave the TODO.